src/Parser.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace CSVParser;
+
+class Parser {
+
+    var $path;
+    var $header_array = array();
+    var $delimiter;
+
+    function __construct ($path, $header_array = array(), $delimiter = ",") {
+
+        if(file_exists($path)){ //verify path
+            $this -> $path = $path; //fill path

```

Comment:
The GitHub Issue required you throw an exception if the filepath specified does not point to an existing file.

You need something like 
```php
if (!file_exists($path))
    throw new Exception("{$path} is not a valid file.");
```

Reply:
I didn't know how to do this.  It's why there is the else statement later.